bootstrap.sh

```diff
@@ -13,6 +18,6 @@ cat <<EOF
 # We support RISC-V, but the default is x86_64 (which we call amd64 for historical reasons):
 export ARCH=amd64
 # And build:
-./util/build
+./util/build -v //:kernel
```

Comment:
nope. you can't do this yet  because blaze can't replace build. 

for now, blaze should be additive, it should not remove anything already there. This command won't produce a working tree, and the other command line will.